demo.md

```diff
@@ -90,11 +90,11 @@ helm install stable/minio \
 
 Login to the Minio UI using a web browser (port 9000) after exposing obtaining the external IP using `kubectl`.
 ```
-kubectl get service argo-artifacts-mini -o wide
+kubectl get service argo-artifacts -o wide
 ```
 On Minikube:
 ```
-minikube service --url argo-artifacts-mini
+minikube -n argo service --url argo-artifacts
```

Comment:
I'm not sure if we need `-n argo` only for minikube. How about the `kubectl` case above?